governance/approveDeny/DEBUG.md

```diff
@@ -160,7 +160,7 @@ Remove Artifactory docker container:
 Sometimes you have to go digging around the Artifactory docker container:
 
   ```
-  docker exec -it --user root rt1 sh
+  docker exec -it --user root rt1 bash
```

Comment:
why?

Reply:
Convenient, mainly. Autocomplete is nice when navigating directories 😄